cluster/saltbase/salt/cluster-autoscaler/cluster-autoscaler.manifest

```diff
@@ -0,0 +1,58 @@
+{% set params = pillar['autoscaler_mig_config'] -%}
+{
+    "kind": "Pod",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "cluster-autoscaler",
+        "namespace": "kube-system",
+        "labels": {
+            "tier": "cluster-management",
+            "component": "cluster-autoscaler"
+        }
+    },
+    "spec": {
+        "hostNetwork": true,
+        "containers": [
+            {
+                "name": "cluster-autoscaler",
+                "image": "gcr.io/mwielgus-proj/cluster-autoscaler:v0.0.1-alpha2-4",
+                "command": [
+                    "./cluster-autoscaler",
+                    "--kubernetes=http://127.0.0.1:8080?inClusterConfig=f",
+                    {% for param in params.split(" ") %}
+                    "{{param}}",
+                    {% endfor %}
+                    "-v=4"
+                ],
+                "resources": {
+                    "limits": {
+                        "cpu": "100m",
+                        "memory": "300Mi"
+                    },
+                    "requests": {
+                        "cpu": "50m",
+                        "memory": "300Mi"
+                    }
+                },
+                "volumeMounts": [
```

Comment:
Some containers don't come with any or uptodate certificate chain. This can make certain interactions with the outside (e.g. talking to googleapis.com) fail. We get around this by mounting in the hosts certificate chain.